fe/spark-dpp/src/main/java/com/starrocks/load/loadv2/dpp/StarrocksKeys.java

```diff
@@ -0,0 +1,38 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Please change the license header

Reply:
The license header has been changed, thanks!